tools/env-loader/pkg/envloader.go

```diff
@@ -0,0 +1,126 @@
+package envloader
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/gravitational/shared-workflows/tools/env-loader/pkg/loaders"
+	"github.com/gravitational/trace"
+)
+
+// This is the path, relative to the git repo root, where environment
+// directories can be found.
+const CiDirectoryRelativePath = "environments"
+
+func findGitRepoRoot() (string, error) {
```

Comment:
Not sure if there's much value in having the tool search for repo root since it could be passed in as an argument. If the tool was less reliant on the git repository structure it I think it would also be more flexible in its usage. (i.e. `environments` config in subfolders, config passed into containers)

I have seen this pattern in some of our other tooling though so maybe it's a common use case. In which case it would maybe be better somewhere in `libs`.

Reply:
Maybe I could make this an arg, and default to git root if not provided? This might make it more flexible while providing a sane default. WDYT?